app/src/androidTest/java/org/akvo/flow/util/HttpUtilTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2010-2017 Stichting Akvo (Akvo Foundation)
+ *
+ * This file is part of Akvo FLOW.
+ *
+ * Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ * the GNU Affero General Public License (AGPL) as published by the Free Software Foundation, either version 3 of the License or any later version.
```

Comment:
This copyright header is outdated, Stellan has made some changes to it and it now looks like in this file:
https://github.com/akvo/akvo-flow-mobile/blob/develop/app/src/main/java/org/akvo/flow/service/ApkUpdateService.java